doc/2/api/controllers/auth/check-rights/index.md

```diff
@@ -0,0 +1,86 @@
+---
+code: true
+type: page
+title: checkRights
+---
+
+# checkRights
+
+<SinceBadge version="change-me"/>
+
+Checks if the provided API request can be executed.
+
+---
+
+## Query Syntax
+
+### HTTP
+
+```http
+URL: http://kuzzle:7512/_checkRights
+Method: POST
+Body:
+```
+
+```js
+{
+  "request": {
```

Comment:
Is this sublevel necessary?

Reply:
Not necessarily, I wanted to be clear that users should pass a request but I don't have string feeling about having the sublevel or not